docs/beta/grain.md

```diff
@@ -1,30 +1,44 @@
+#### 1. Cred Calculation
+In this cycle, the project recomputes the Cred Contribution Graph by gathering the data your plugins are tracking, and calculating new Cred scores. By default, this cycle runs every 6 hours.
+
+#### 2. Grain Distribution
+In this less-frequent cycle, the project distributes Grain based on participants’ newest Cred scores. By default, this cycle runs once a week.
+
+Your Cred score determines the percentage of Grain you receive out of the total amount allocated to the community for each Harvest, regardless of how frequently Cred Calculation happens within it. 
+
+#### Grain Distribution Policies
+SourceCred currently has two policies for how a project distributes Grain:
+
+- `IMMEDIATE` splits Grain evenly based on everyone’s Cred earned per the most recent Grain Distribution cycle. (default: last week)
+- `BALANCED` splits Grain based on lifetime Cred. This will accommodate a retroactively-updated Cred shift by equalizing compensation as soon as possible, using as few Harvests as possible to achieve lifetime distribution balance. Note: This means that overpaid contributors might receive 0 Grain for some period of time, if it takes more than one full Harvest worth of Grain distribution to achieve balance.
 
 
 ### _“What if I contributed a lot before a small Grain distribution, and then less before a bigger Grain distribution?”_
 
 Since Cred is retroactive, the SourceCred algorithm will make sure your percentage is based on _all_ of your contributions compared to all existing contributions, and make sure your percentage is based on the total amount of Grain that has been issued to the community. _Ever._ 
 
+### _“Can I buy Grain?”_
 
-You can also get Grain by becoming a sponsor. Buying Grain sponsors contributors directly (by paying them for their Grain). Thus, anyone holding Grain has supported the project, either directly through contributions, or indirectly through financial sponsorship. When someone wants to sell their Grain for dollars (or some other currency), the money either comes from sponsors buying their Grain, or a pool their community has previously set up.
+Yes, you can also get Grain by becoming a sponsor. Buying Grain sponsors participants directly (by paying them for their Grain). Thus, anyone holding Grain has supported the project, either directly through contributions, or indirectly through financial sponsorship. When someone wants to sell their Grain for dollars (or some other currency), the money either comes from sponsors buying their Grain, or a pool their community has previously set up.
```

Comment:
Hmm, this isn't accurate in general (or for SourceCred). This is up to the project to decide. Currently in SourceCred you can sell Grain but you can't buy it.

Reply:
Okay. That's mostly all coming from how the old doc was before I came to the project, so now I'm confused… I've been told repeatedly that you can buy Grain $1:1 but I guess that's always been with the unspoken caveat of "in the future, once we implement that as an option" ?